### ISSUE TITLE
[BUG] "An error occured" on modal on /timesheets/showAll (2.4-beta-4)

### ISSUE THREAD

**Issue by steveoriol** [Leantime/leantime#1910]
**What set up are you using**
* [ ] Cloud Hosted Version
* [ x] Self Hosted

**Describe the bug**
Get an error modal window with "An error occured" on the "All Timesheets" page when you try to click on an All Timesheets entity for edit it  exeample : "#\1 - Edit" (1st colunn)  '/timesheets/editTime/1'

**To Reproduce**
1. Add a task and the time spent on it using the "Time tracking" tab.
2. Go to the "All Timesheets" page (/timesheets/showAll)
3. Click on "#\1 - Edit" (1st colunn)
4. See error the modal error

**Expected behavior**
can edit the entity, change time, date or even delete it....

**Leantime Version**
2.4-beta-4

**Server & PHP / MySQL Version**
docker compose version : https://github.com/Leantime/docker-leantime

**Comment by jiriks74** [comment]
It looks like it's fixed:
![image](https://github.com/Leantime/leantime/assets/54378412/7cc1f0d2-af82-4a01-8e51-be3f1e675402)

It may have been fixed in #1914 ?

**Comment by jiriks74** [comment]
If you want a quick fix until the Docker image is updated run this in your docker container:
```bash
docker exec -it <container_name> /bin/sh
```

```sh
wget https://gist.githubusercontent.com/jiriks74/826137927d216e248c0b35b6cc5349c9/raw/003aa74f25a346024004f0a3600fc6bc0aeca8e0/timesheets.patch
apk add git
git apply timesheets.patch
rm -f timesheets.patch
```

**Comment by marcelfolaron** [comment]
just released 2.4-beta-6 which should fix this
